sharding/tests/test_validator_manager_utils.py

```diff
@@ -27,62 +25,47 @@
 configure_logging(config_string=config_string)
 '''
 
-# Testing Part
-def deploy_tx(state, tx):
-    success, output = apply_transaction(state, tx)
-    if not success:
-        raise t.TransactionFailed("Failed to deploy tx")
-    return output
-
-
-def deploy_contract(state, sender_privkey, bytecode):
-    tx = Transaction(
-            state.get_nonce(utils.privtoaddr(sender_privkey)),
-            GASPRICE, STARTGAS, to=b'', value=0,
-            data=bytecode
-    ).sign(sender_privkey)
-    return deploy_tx(state, tx)
-
-
-def deploy_initializing_contracts(sender_privkey, state):
-    sender_addr = utils.privtoaddr(sender_privkey)
-    txs = mk_initiating_contracts(sender_privkey, state.get_nonce(sender_addr))
-    for tx in txs:
-        try:
-            deploy_tx(state, tx)
-        except t.TransactionFailed:
-            pass
 
 num_blocks = 6
 
-@pytest.fixture
+
+@pytest.fixture()
```

Comment:
Is there any difference between `@pytest.fixture` and `@pytest.fixture()`?
I saw the usage of `@pytest.fixture` in the [pytest doc](https://docs.pytest.org/en/latest/fixture.html#fixtures-as-function-arguments). Though `@pytest.fixture()` is used as well.